nservicebus/logging/usage.md

```diff
@@ -1,12 +1,12 @@
 ---
 title: Logging from the user code
-reviewed: 2019-07-29
+reviewed: 2021-09-17
 component: Core
 redirects:
 - nservicebus/logging-writing
 ---
 
-The NServiceBus logging abstractions can be used for writing log messages from the user code. This approach ensures that both NServiceBus and user code log messages are written to the same destinations.
+While the use of [`Microsoft.Extensions.Logging`](/nservicebus/logging/extensions-logging.md) is recommended for new endpoints the NServiceBus logging abstraction can also be used for writing log messages from user code.
```

Comment:
Do you want to say that this is enabled when using Generic Host with NServiceBus.Extensions.Hosting, or when self-hosting and using NServiceBus.Extensions.Logging? Or something to say you can use one or the other but not both?

Reply:
yea, I've made some tweaks, see if you like it?